sql-plugin/src/main/scala/org/apache/spark/sql/rapids/aggregate/aggregateFunctions.scala

```diff
@@ -606,17 +607,31 @@ case class GpuFloatMax(child: Expression) extends GpuMax(child)
 case class GpuExtractChunk32(
     data: Expression,
     chunkIdx: Int,
-    replaceNullsWithZero: Boolean) extends GpuExpression with ShimExpression {
+    replaceNullsWithZero: Boolean,
+    isForDecimal: Boolean) extends GpuExpression with ShimExpression {
```

Comment:
NIT: It is missing in the doc above.
```
* @param isForDecimal XXXXxXXX
```

And looks like it may extract a 32-bit chunk from a 64-bit value if isForDecimal is false. So the doc needs to be updated.